proto/cosmos/epochs/v1beta1/query.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+package cosmos.epochs.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/epochs/v1beta1/genesis.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/epochs/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // EpochInfos provide running epochInfos
+  rpc EpochInfos(QueryEpochInfosRequest) returns (QueryEpochInfosResponse) {
+    option (google.api.http).get = "/cosmos/epochs/v1beta1/epochs";
+  }
+  // CurrentEpoch provide current epoch of specified identifier
+  rpc CurrentEpoch(QueryCurrentEpochRequest) returns (QueryCurrentEpochResponse) {
+    option (google.api.http).get = "/cosmos/epochs/v1beta1/current_epoch";
+  }
+}
+
+// QueryEpochInfosRequest defines the gRPC request structure for
+// querying all epoch info.
+//
+// Since: cosmos-sdk 0.53
```

Comment:
i don't think we need the since comments here - this would imply that epochs proto existed in some capacity in a previous release (i.e. v0.50, v0.47..) and has since been updated with these additional messages

Reply:
My thinking is that this just says when this proto message was introduced which is how it is used in the other parts of the protos

Reply:
i don't think thats how its used in other protos - you only see `Since` when additions are made to an existing package

Reply:
x/group actually uses `Since` at the top of the package, perhaps we should consider doing that. but i dont think message level `Since` makes sense here https://github.com/cosmos/cosmos-sdk/blob/bc1aa0645c6f0303b893223d568016c8b48b0002/proto/cosmos/group/v1/tx.proto#L1

Reply:
[747fad8](https://github.com/cosmos/cosmos-sdk/pull/23815/commits/747fad88a9c99bdeeafdaa9fae61a5ce7d0a88e5)